<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>200</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="registerButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backToLoginButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>To Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstNameText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your first name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondNameText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your second name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="loginText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Enter your new password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>180</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your email</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;First name, login and&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;password are required&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
